### DISCUSSION TITLE
Welcome to nano Discussions!

### DISCUSSION THREAD

**Discussion by Nebelung-Dev** [nano-proxy/nano#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by code-leech** [comment]
This is the best proxy that I've found because it ACTUALLY WORKS

I've been tweaking a few projects to work in my school, unfortunately, the blocker is a firewall called "NGFL/LGFL". It's really tough and seems to not like certain service workers. This one works most of the time, but maybe something can be worked out?

I don't know much about Ultraviolet code and JS

Thanks

**Comment by RombotLabs** [comment]
Hello together!
I have a short question about nano, how much does nano increases the data traffic?
Thanks for answering!
Bye
